commands/settings/index.js

```diff
@@ -26,7 +26,7 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'pg',
   command: 'settings',
-  description: 'show current settings',
+  description: 'shows your database current postgres settings values',
```

Comment:
Shall I compare thee to `log_lock_waits`? Thou art more lovely and more temperate.